config/vufind/Demo.ini

```
@@ -236,6 +236,16 @@ extraHoldFields = "comments:requestGroup:pickUpLocation:proxiedUsers:startDate:r
 ; Should we include a link to a fake OPAC?
 fakeOpacLink = false
 
+; Block ILS methods during given times. Format is "method:setting = value"
+; method   - Method in ILS driver
+; settting - Valid settings are startDate, endDate, recurringStart, recurringEnd
+; value    - Date for startDate/endDate, time for recurringStart/recurringEnd
+;[TimedBlocks]
+;Renewals:startDate = 2025-07-01
+;Renewals:endDate = 2025-08-01
+;Renewals:recurringStart = 01:00
+;Renewals:recurringEnd = 03:00
```

If you formatted this like:

```suggestion
;Renewals[startDate] = 2025-07-01
;Renewals[endDate] = 2025-08-01
;Renewals[recurringStart] = 01:00
;Renewals[recurringEnd] = 03:00
```

...I think you could simplify your config parsing a bit, since you'd have a formatted array right out of the .ini file without needing to split keys and rearrange data.

---

Now simplified.